src/status_im/ui/screens/activity_center/notification/mentions/view.cljs

```diff
@@ -0,0 +1,55 @@
+(ns status-im.ui.screens.activity-center.notification.mentions.view
+  (:require [quo.components.animated.pressable :as animation]
+            [quo2.core :as quo2]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.multiaccounts.core :as multiaccounts]
+            [status-im.ui.screens.activity-center.notification.mentions.style :as style]
+            [status-im.ui.screens.activity-center.utils :as activity-center.utils]
+            [status-im.utils.datetime :as datetime]
+            [utils.re-frame :as rf]
+            [clojure.string :as str]))
+
+(defn message-body [message]
+  (let [parsed-text (get-in message [:content :parsed-text])
+        parsed-text-children (:children (first parsed-text))]
+    [text/text {:number-of-lines     2
+                :style               style/tag-text
+                :accessibility-label :activity-message-body
+                :size                :paragraph-1}
+     (for [{:keys [type literal]} parsed-text-children]
+       (case type
+         "mention" [text/text {:style style/mention-text
+                               :size  :paragraph-1}
+                    (str "@" (rf/sub [:contacts/contact-name-by-identity literal]))]
```

Comment:
wow that looks confusing at first, like deref of sub :)

Reply:
Yeah :)